pages/dmp_supporting_info/support_00_local_disc.md

```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
 
 Helpdesk: [helpdesk@gbif.no](https://www.gbif.no/services/ "2024-07-19")
 
+### NALMIN - Norwegian Advanced Light Microscopy Imaging Network
+Norwegian Advanced Light Microscopy Imaging Network (NALMIN) principal goal is to establish a network at international competence level for advanced light microscopy. The network consists of 5 national nodes located in Oslo (1 node with two branches(NorMIC IBV and NorMIC Radium)), Bergen, Trondheim (2 nodes) and Tromsø, each with specialized competence. All nodes are members of the national NorBioimaging infrastructure organization, and the NorMIC Oslo node is a member of the [EuroBioimaging ECRIN](https://www.eurobioimaging.eu/nodes/normic-oslo---advanced-light-microscopy-node-oslo) network. 
```

Comment:
Are the node specifics really necessary?

Reply:
I think this is helpful, since most of the support is provided locally.